### ISSUE TITLE
Issue when using a non-standard docker-ce repository URL

### ISSUE THREAD

**Issue by matt6697** [weareinteractive/ansible-docker#9]
The following tasks fail when the ansible module is used on a network not having direct access to `download.docker.com` (local debian mirror used with internal URL).

```
- name: Adding APT key
  apt_key:
    id: 0EBFCD88
    url: https://download.docker.com/linux/{{ ansible_distribution | lower }}/gpg

- name: Adding APT repository
  apt_repository:
    repo: "deb [arch=amd64] https://download.docker.com/linux/{{ ansible_distribution | lower }} {{ ansible_distribution_release }} stable"
    update_cache: yes
```

Whould it be possible to use a `docker_ce_repository_baseurl `variable defaulted to `https://download.docker.com/linux/` instead ? Debian/Ubuntu example below.

```
- name: Adding APT key
  apt_key:
    id: 0EBFCD88
    url: "{{ debian_docker_ce_private_repo_gpg }}"

- name: Adding APT repository
  apt_repository:
    repo: "deb [arch=amd64] {{ debian_docker_ce_private_repo }} {{ ansible_distribution_release }} stable""
    update_cache: yes
```

**Comment by matt6697** [comment]
PR https://github.com/weareinteractive/ansible-docker/pull/10 proposed to make use of a private mirror of the official docker-ce repository possible.

**Comment by matt6697** [comment]
Issue resolved with PR #10 merge. Thanks ;-)
